### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by tmachyta**

### CODE REVIEW

File: src/main/java/booktopiahub/controller/CategoryController.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/categories")
+public class CategoryController {
+    private final CategoryService categoryService;
+
+    @GetMapping
+    @Operation(summary = "Get all categories",
+            description = "Get a list of all available categories")
+    public List<CategoryDto> findAll(Pageable pageable) {
```
Comment (olekskov) [inline src/main/java/booktopiahub/controller/CategoryController.java]: I don't think swagger will work correctly with pagable without this annotation.
Please recheck and correct in all controllers if I'm correct
```suggestion
    public List<CategoryDto> findAll(@ParameterObject Pageable pageable) {
```

**Review by olekskov — CHANGES_REQUESTED** [review]

File: src/main/java/booktopiahub/controller/CategoryController.java (hunk 1 above)
Comment (tmachyta) [reply src/main/java/booktopiahub/controller/CategoryController.java]: Updated

Comment (tmachyta) [reply src/main/java/booktopiahub/controller/CategoryController.java]: Also added requirements according roles

**Review by Ivan95kos — APPROVED** [review]
-

**Review by Ivan95kos — CHANGES_REQUESTED** [review]
add the SECURITY REQUIREMENTS

**Comment by tmachyta** [comment]
> add the SECURITY REQUIREMENTS

But, I have already added  @PreAuthorize("hasRole('USER')") and  @PreAuthorize("hasRole('ADMIN')") - annotations in controllers

**Review by Ivan95kos — APPROVED** [review]
Great job

**Pull request merged**